indico/modules/events/logs/controllers.py

```diff
@@ -29,17 +31,20 @@ class RHEventLogs(RHManageEventBase):
     """Shows the modification/action log for the event"""
 
     def _process(self):
+        metadata_query = {k.lstrip('meta.'): int(v) if v.isdigit() else v
```

Comment:
`lstrip` doesn't work this way. it removes all of the chars, so `meta.event_id` would turn into `vent_id`.. Only python 3.9 has a function to strip a prefix, so for now I'd keep the slice you had before..